db.go

```diff
@@ -92,30 +93,43 @@ func (db *Database) Create() error {
 	return nil
 }
 
-func (db *Database) GetView(docName string, viewName string) (error, []byte) {
+func (db *Database) GetView(docName string, viewName string, key string) ([]byte, error) {
+
+	log.Print("couch : GetView key " + key + " in viewName " + viewName + " of desDoc " + docName)
```

Comment:
I think a formatted string would be easier to grok. https://godoc.org/github.com/Sirupsen/logrus#Printf